src/app/_components/TagComponents/TagGroup.tsx

```diff
@@ -0,0 +1,34 @@
+import { type TagProps, TagLabel } from './TagLabel'
+
+type TagItem = {
+  variant?: TagProps['variant']
+  text: TagProps['children']
+  icon?: TagProps['icon']
+}
+
+export type TagGroupProps = {
+  tags: [TagItem] | [TagItem, TagItem]
+}
+
+export function TagGroup({ tags }: TagGroupProps) {
+  if (tags.length === 1) {
+    const { text, icon } = tags[0]
+    return <TagLabel icon={icon}>{text}</TagLabel>
+  }
+
+  return (
+    <span className="flex gap-2">
+      {tags.map(({ variant, text, icon }, index) => (
+        <TagLabel
+          key={index}
+          icon={icon}
+          variant={
+            index === 0 ? 'primary' : index === 1 ? variant : 'secondary'
```

Comment:
```suggestion
            index === 0 ? 'primary' : (variant ?? 'secondary')
```

Reply:
![Screenshot 2024-12-19 at 14 39 03](https://github.com/user-attachments/assets/b4976b0d-da5e-4753-bd3e-f3672b917297)

Reply:
AH yes. 😵‍💫 Thank you! ❤️